common/changes/office-ui-fabric-react/datePicker_textField_className_2019-01-17-19-21.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "added className attribute to TextField of DatePicker",
```

This comment is used for official release notes. I would suggest something like this:
```suggestion
      "comment": "DatePicker: adds missing className attribute to the `TextField` of the control.",
```